packages/graphql-server/src/resolvers/appointment/api.ts

```diff
@@ -0,0 +1,115 @@
+import { fetcher, setQueryParams } from '@reapit/elements'
+import logger from '../../logger'
+import { ServerContext } from '../../app'
+import errors from '../../errors'
+import { API_VERSION } from '../../constants/api'
+import {
+  GetAppointmentsArgs,
+  GetAppointmentByIdArgs,
+  UpdateAppointmentArgs,
+  CreateAppointmentArgs,
+  GetAppointmentsResult,
+  Appointment,
+} from './appointments'
+
+export const URLS = {
+  appointments: '/appointments',
+}
+
+export const callGetAppointmentsAPI = async (args: GetAppointmentsArgs, context: ServerContext) => {
+  const traceId = context.traceId
+  try {
+    logger.info('callGetAppointmentsAPI', { traceId, args })
+
+    const createResponse = await fetcher({
+      url: `${URLS.appointments}/?${setQueryParams(args)}`,
+      api: process.env['PLATFORM_API_BASE_URL'],
+      method: 'GET',
+      headers: {
+        authorization: context.authorization,
+        'Content-Type': 'application/json',
+        'api-version': API_VERSION,
+      },
+      body: {},
+    })
+    return createResponse as GetAppointmentsResult
+  } catch (error) {
+    logger.error('callGetAppointmentsAPI', { traceId, error })
+    return errors.generateUserInputError(traceId)
+  }
+}
+export const callGetAppointmentByIdAPI = async (args: GetAppointmentByIdArgs, context: ServerContext) => {
+  const traceId = context.traceId
+  logger.info('callGetAppointmentByIdAPI', { args, traceId })
+  try {
+    const response = await fetcher({
+      url: `${URLS.appointments}/${args.id}`,
+      api: process.env['PLATFORM_API_BASE_URL'],
+      method: 'GET',
+      headers: {
+        authorization: context.authorization,
+        'Content-Type': 'application/json',
+        'api-version': API_VERSION,
+      },
+    })
+    return response as Appointment
+  } catch (error) {
+    logger.error('callGetAppointmentByIdAPI', { traceId, error })
+    return errors.generateUserInputError(traceId)
+  }
+}
+
+export const callUpdateAppointmentByAPI = async (args: UpdateAppointmentArgs, context: ServerContext) => {
+  const traceId = context.traceId
+  logger.info('callUpdateAppointmentByAPI', { args, traceId })
+  try {
+    await fetcher({
```

Comment:
const updateResponse = ....

Reply:
Update appointment API is currently responding HTTP status 201 with no response if executing successfully